docs/faq/contributing.txt

```diff
@@ -107,3 +107,35 @@ I'm sure my ticket is absolutely 100% perfect, can I mark it as "Ready For Check
 
 Sorry, no. It's always better to get another set of eyes on a ticket. If
 you're having trouble getting that second set of eyes, see questions above.
+
+Tips to help your PR get reviewed faster
+========================================
+
+1. Verify your PR is in the review queue.
+
+   Make sure your pull request appears in the `review queue`_ so reviewers can
+   easily find it.
+
+2. Find potential reviewers.
+
+   Look for contributors who are familiar with the component you're working on
+   and are still active.
+   You can do this by filtering *Ready for checkin* tickets by component and
+   grouping them by owner to see recent activity.
+   For example, for GIS-related changes, use the `GIS component filters`_ to
+   find active contributors.
```

Comment:
I don't know about this suggestion. I personally wouldn't want to be pinged with review requests. I'd rather have some list of people who want to be pinged (if any exist) so that this isn't a research activity for each contributor.